src/gui/attributetable/qgsattributetablemodel.cpp

```diff
@@ -40,6 +41,7 @@ QgsAttributeTableModel::QgsAttributeTableModel( QgsVectorLayerCache *layerCache,
     , mLayerCache( layerCache )
     , mFieldCount( 0 )
     , mCachedField( -1 )
+    , mLoadWorker( NULL )
```

Comment:
If a null representation should be used in C++ it's better to use `nullptr` which is actually defined in the standard.